conf/layer.conf

```diff
@@ -9,4 +9,4 @@ BBFILE_PATTERN_go-binary-layer := "^${LAYERDIR}/"
 BBFILE_PRIORITY_go-binary-layer = "7"
 
 LAYERDEPENDS_go-binary-layer = "core"
-LAYERSERIES_COMPAT_go-binary-layer = "dunfell kirkstone mickledore nanbield"
+LAYERSERIES_COMPAT_go-binary-layer = "sumo dunfell kirkstone mickledore nanbield"
```

Comment:
@kraj hopefully adding sumo back in is OK -- it seems to be working OK here in a build.

Reply:
yes should be fine, I think this is the purpose of this binary layer anyway

Reply:
meta-go-binary is working great -- too well in fact -- I have very little incentive to update the base OS :frowning_face: